src/subgraph/actions/listSales.ts

```diff
@@ -21,15 +21,18 @@ export const listSales = async <T>(
   }
 
   const sales: TokenSale[] = queryResult.data.tokenSales.map((e) => {
-    return {
+    const sale: TokenSale = {
```

Comment:
Consider refactoring the code in `listAllSales.ts` and `listSales.ts` to use the same helper function to map from subgraph dto => Token Sale